package/shell-setup.sh

```diff
@@ -15,9 +16,13 @@ SERVER="${CATTLE_SERVER}/k8s/clusters/${cluster}"
 if [ -f /etc/kubernetes/ssl/certs/serverca ]; then
     cp /etc/kubernetes/ssl/certs/serverca .kube/certs/ca.crt
     chmod 666 .kube/certs/ca.crt
-    if ! curl -s $SERVER > /dev/null; then
-        CERT="    certificate-authority: /nonexistent/.kube/certs/ca.crt"
-    fi
+else
+    echo ${cacerts} | base64 -d > .kube/certs/ca.crt
+    chmod 666 .kube/certs/ca.crt
+fi
+
+if ! curl -s $SERVER > /dev/null; then
```

Comment:
to keep old behavior, do `if [ -f /nonexistent/.kube/certs/ca.crt ] && ! curl -s $SERVER > /dev/null` here